components/google_recaptcha/app/google_recaptcha.app.ts

```diff
@@ -1,13 +1,25 @@
 import { defineApp } from "@pipedream/types";
+import { axios } from "@pipedream/platform";
 
 export default defineApp({
   type: "app",
   app: "google_recaptcha",
   propDefinitions: {},
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    async validateRecaptcha({
+      $,
+      params,
+    } = {
+      params: {},
+      $: this,
+    }) {
+      axios($, {
```

Comment:
Hello @zalmanlew, we missed the `return` here that causing the API call response skipped. Here's my suggestion
```suggestion
      return axios($, {
```